src/app/loaders/express/index.ts

```diff
@@ -11,11 +11,10 @@ import { AuthRouter } from '../../modules/auth/auth.routes'
 import { ExamplesRouter } from '../../modules/examples/examples.routes'
 import { AdminFormsRouter } from '../../modules/form/admin-form/admin-form.routes'
 import { PublicFormRouter } from '../../modules/form/public-form/public-form.routes'
-import { FrontendRouter } from '../../modules/frontend/frontend.routes'
-import * as HomeController from '../../modules/home/home.controller'
```

Comment:
should we delete `home.controller` and `home.routes` (basically the whole `home` folder) too?

Reply:
makes sense! done in [5c99bc0](https://github.com/opengovsg/FormSG/pull/6190/commits/5c99bc0413c13d3ffd57680cdfc20a01e4610487)